### PR TITLE
Delete orphans or add to TOC

### PR DESCRIPTION
**Pull request by gewarren**
Contributes to #27892.

[Preview](https://review.docs.microsoft.com/en-us/dotnet/orleans/overview?branch=pr-en-us-28198).

For the Orleans orphans, I added all of the orphaned articles to the TOC except for hello-world.md, because there already appears to be a separate Hello World tutorial.

### CODE REVIEW

File: includes/book-preview.md [hunk 1]
```diff
@@ -1,4 +0,0 @@
-> [!IMPORTANT]
```
Comment (gewarren) [inline includes/book-preview.md:1]: Perhaps this will be reused in the future?

**Comment by gewarren** [comment]
@adegeo has OK'd the upgrade assistant images.

File: docs/orleans/toc.yml [hunk 2]
```diff
@@ -1,12 +1,17 @@
+items:
```
Comment (adegeo) [inline docs/orleans/toc.yml:1]: This whole file's indentation isn't uniform.

File: docs/orleans/tutorials-and-samples/adventure.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
-# Adventure sample project
+# Adventure game sample project
 
-A simple multiplayer text adventure game inspired by old-fashioned, text-based adventure games.
+This sample is a simple multiplayer text adventure game inspired by old-fashioned, text-based adventure games.
 
 ## Instructions
 
-1. Open _OrleansAdventure.sln_ in Visual Studio. Found [here.](https://github.com/dotnet/orleans/tree/main/samples/Adventure)
+1. Open [OrleansAdventure.sln](https://github.com/dotnet/orleans/tree/main/samples/Adventure) in Visual Studio.
```
Comment (adegeo) [inline docs/orleans/tutorials-and-samples/adventure.md:13]: Should you still use _italic_ for file? Unsure
```suggestion
1. Open [_OrleansAdventure.sln_](https://github.com/dotnet/orleans/tree/main/samples/Adventure) in Visual Studio.
```

File: includes/book-preview.md (hunk 1 above)
Comment (adegeo) [reply includes/book-preview.md:1]: I wouldn't be opposed to keeping it.

**Review by adegeo — CHANGES_REQUESTED** [review]
Some minor things to take care of

File: docs/orleans/toc.yml (hunk 2 above)
Comment (gewarren) [reply docs/orleans/toc.yml:1]: I saw that, but it's rendering fine, and I don't know an easy way to fix it.

**Review by adegeo — APPROVED** [review]

**Review by IEvangelist — APPROVED** [review]
Thank you @gewarren, I was leaving the images in place originally as I was going to redo all of them. I wanted to redo them incrementally, and have the old ones in place so that I knew what I was replacing.

**Pull request merged**